web-client/src/modules/timeline/containers/TimelineViewContainer/TimelineViewContainer.tsx

```diff
@@ -8,12 +8,14 @@ import { setOffer } from 'src/ducks/offers/actions';
 import { OffersState } from 'src/ducks/offers/types';
```

Comment:
Changed to query for RequestWithOfferANdTimeline (should really be PostWithOfferAndTimeline).  Commented out code I did not understand.  A lot of it had to do with what to do after viewing timeline.